CHANGELOG.md

```diff
@@ -13,6 +13,13 @@ vNext (Month Day, Year)
 - :bug: Fix an issue where `smithy-xml` may have generated invalid XML (smithy-rs#719)
 - :bug: Fix error when receiving empty event stream messages (smithy-rs#736)
 - :bug: Fix bug in event stream receiver that could cause the last events in the response stream to be lost (smithy-rs#736)
+- :tada: Add support for 6 new AWS services:
+  - Wisdom
+  - VoiceId
+  - Account
+  - KafkaConnect
+  - OpenSearch
+  - CloudControl
```

Comment:
This isn't really related to smithy-rs. Makes sense to have the SDK changelog though.